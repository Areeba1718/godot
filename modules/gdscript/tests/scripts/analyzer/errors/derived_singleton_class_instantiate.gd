class A extends IP:
	pass

func test():
	A.new()
