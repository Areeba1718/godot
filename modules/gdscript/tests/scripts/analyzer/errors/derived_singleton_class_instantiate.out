GDTEST_ANALYZER_ERROR
Class "derived_singleton_class_instantiate.gd::A" cannot be constructed as it is based on singleton native class "IP".
